ledger/error/src/main/scala/com/daml/error/definitions/groups/RequestValidation.scala

```diff
@@ -4,7 +4,6 @@
 package com.daml.error.definitions.groups
 
 import java.time.Duration
-
```

Comment:
If it was not accidentally removed could you please sort it alphabetically?

Reply:
tbh, I never check imports but rely on IJ to sort them out